io.github.jordanl2.ModularCalculator.yml

```
@@ -0,0 +1,30 @@
+app-id: io.github.jordanl2.ModularCalculator
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: modularcalculator
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-documents
```

```suggestion
  - --filesystem=xdg-documents
  - --device=dri
```

This will fix these errors:

```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri, suffix _dri)
libGL error: failed to load driver: amdgpu
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: radeonsi
```